column_reader.go

```diff
@@ -406,8 +417,16 @@ func (r *fixedLenByteArrayColumnReader) ReadValues(values []Value) (n int, err e
 	}
 }
 
-func (r *fixedLenByteArrayColumnReader) Reset(decoder encoding.Decoder) {
+func (r *fixedLenByteArrayColumnReader) Reset(numValues int, decoder encoding.Decoder) {
 	r.decoder = decoder
 	r.buffer = r.buffer[:0]
 	r.offset = 0
+	r.remain = r.size * numValues
```

Comment:
A little confusing reading this that `remain` might be the number of items or the number of bytes depending on the struct type.

Reply:
True. In this context the `offset` was computed as a number of bytes, so I kept the number of remaining items in bytes as well.

If delaying the scaling from items to bytes will add a significant number of multiplications in the read path 🤷